article/2002/meta.json

```diff
@@ -0,0 +1,16 @@
+{
+    "source": "https://www.wired.co.uk/article/vint-cerf-interplanetary-internet",
+    "title": "Father of the internet, Vint Cerf, on creating the interplanetary internet",
+    "author": "Adam Mann",
+    "images": [
+        {
+            "file": "1.jpg",
+            "credit": "NASA/JPL"
+        },
+        {
+            "file": "2.jpg",
+            "caption": "Father of the Internet Vint Cerf is responsible for helping develop the TCP/IP protocols that underly the web. In his role as Google’s chief internet evangelist, Cerf is dedicated to thinking about the future of the net, including its use in space",
+            "credit": "Guido van Nispen/Flickr/CC BY 2.0"
+        }
```

Comment:
For these images, it doesn't show anywhere in the article where they are supposed to be rendered. I think we could use markdown to render these as well by adding

```md
![alt image title](/article/2002/1.jpg)
````
instead of having it in the meta. That way we can specify where in the article the image is going to be.

Reply:
@scottjr632 I was actually imagining We can have a carousel widget at the bottom for now that can show the caption and credit below it, what do you think? Like a creative step two after author, title, and article are properly dispalyed.